pkg/utils/k8s_utils.go

```diff
@@ -234,9 +234,9 @@ func SkipTests(skipFile string) (string, error) {
 	if skipFile == "" {
 		skipFile = IgnoreFile
 	}
-	file, err := os.ReadFile(filepath.Clean(skipFile))
+	file, _ := os.ReadFile(filepath.Clean(skipFile))
 	var tests SkippedTests
-	err = yaml.Unmarshal(file, &tests)
+	err := yaml.Unmarshal(file, &tests)
```

Comment:
Same as above comment, here as well.

Reply:
Same as above, the parameter is defined in `k8s_utils_test.go`. we can be sure that it won't panic